mentorship ios/Views/Registration/Login.swift

```diff
@@ -68,9 +59,34 @@ struct Login: View {
                     SignUp(isPresented: self.$showSignUpPage)
                 }
             }
-
-            //spacer to push content to top
-            Spacer()
+            
+            //activity indicator or show user message text
+            if self.loginViewModel.inActivity {
+                ActivityIndicator(isAnimating: $loginViewModel.inActivity)
+            } else if !(self.loginViewModel.loginResponseData.message?.isEmpty ?? true) {
+                Text(self.loginViewModel.loginResponseData.message ?? "")
+                    .modifier(ErrorText())
+            }
+            
+            // Divider for social sign in options
+            ZStack {
```

Comment:
Shouldn't this be in Vstack?

Reply:
This ZStack (used to shown "Or" text on top of divider) is inside VStack.